src/views/scheduleMessage.js

```diff
@@ -1,13 +1,14 @@
 import './scheduleMessage.css';
 
-import MessageInput from '../components/messageInput/messageInput'
+import MessageInput from '../components/messageInput/messageInput';
+import TimePicker from '../components/timepicker/timePicker'
 import { useContext, useEffect, useRef, useState } from 'react';
```

Comment:
Remove unused import
```suggestion
import { useContext, useEffect, useState } from 'react';
```